demos/projects/ESPRESSIF/az-nvs-cert-bundle/main/az_nvs_cert_bundle_main.c

```diff
@@ -96,183 +97,216 @@ static uint8_t trust_bundle[] =
     "AHFiRSdLOkKEW39lt2c0Ui2cFmuqqNh7o0JMcccMyj6D5KbvtwEwXlGjefVwaaZB\r\n"
     "RA+GsCyRxj3qrg+E\r\n"
     "-----END CERTIFICATE-----\r\n";
-static uint32_t trust_bundle_size = sizeof( trust_bundle ) - 1;
-static uint32_t trust_bundle_version = 1;
+static uint32_t ulTrustBundleSize = sizeof( ucTrustBundle ) - 1;
+static uint8_t ucTrustBundleVersion[] = "1.0";
 
 /*
  *  Print trust bundle saved in the NVS
  */
-esp_err_t print_saved_bundle( nvs_handle_t my_handle )
+esp_err_t prvPrintSavedBundle( nvs_handle_t xNVSHandle )
```

Comment:
This should be moved within the trust_bundle_storage compilation unit and reuse NVS code instead of duplicating it.